src/chigame/games/views.py

```diff
@@ -357,6 +357,11 @@ def form_valid(self, form):
     def get_success_url(self):
         return reverse_lazy("tournament-detail", kwargs={"pk": self.object.pk})
 
+    def create_chat(self):
+        # Auto-create a chat for this respective tournament
+        chat = Chat(tournament=self)
```

Comment:
This function isn't working right now, and no chat is being created. You need to create the chat inside the form_valid function. Please make sure to test that this function is actually creating a chat before you push again!

Reply:
This function was creating a new chat each time on my end, but I will double check to see what is going on.